salt/states/pkg.py

```diff
@@ -80,10 +80,11 @@
 import os
 import re
 
-# Import salt libs
-import salt.utils
+# Import Salt libs
+import salt.utils  # Can be removed once gen_state_tag is moved
 import salt.utils.pkg
 import salt.utils.platform
+import salt.utils.versions
```

Comment:
I missed making an update to one of the refs to `salt.utils.compare_versions`. Just pushed a commit to address this. Good catch!